common/src/main/java/com/revenuecat/purchases/common/Config.kt

```diff
@@ -4,5 +4,5 @@ object Config {
 
     var debugLogsEnabled = false
 
-    const val frameworkVersion = "4.3.0-SNAPSHOT"
+    const val frameworkVersion = "4.3.0-beta1"
```

Comment:
there shouldn't be any issues with the version numbers here, right? I'm asking because on iOS, in some places like `CFBundleVersion` the format needs to be numbers-and-dots only.

Reply:
Should be fine

Reply:
I think this should be `beta.1`

Reply:
you're right, I'd missed [that section in semver](https://semver.org/#spec-item-11), I'll update